packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsEditor.tsx

```diff
@@ -103,9 +103,8 @@ export default function DataAccessRequestAccessorsEditor(
           {accessorChanges.map((ac, i) => {
             return (
               <div className={'list-items'} key={`accessor-${i}`}>
-                <UserCard
-                  size={'SMALL USER CARD'}
-                  ownerId={ac.userId}
+                <UserBadge
+                  userId={ac.userId}
```

Comment:
Deprecated the "SMALL USER CARD" option (but still works since it's used in SWC). Replace all with a new component `UserBadge`